README.md

```diff
@@ -128,18 +128,20 @@ must [supply credentials](#authentication) and a project ID if running this snip
 import com.google.gcloud.bigquery.BigQuery;
 import com.google.gcloud.bigquery.BigQueryOptions;
 import com.google.gcloud.bigquery.Field;
+import com.google.gcloud.bigquery.Job;
 import com.google.gcloud.bigquery.JobStatus;
 import com.google.gcloud.bigquery.JobInfo;
 import com.google.gcloud.bigquery.LoadJobConfiguration;
 import com.google.gcloud.bigquery.Schema;
 import com.google.gcloud.bigquery.StandardTableDefinition;
+import com.google.gcloud.bigquery.Table;
 import com.google.gcloud.bigquery.TableId;
 import com.google.gcloud.bigquery.TableInfo;
 
 BigQuery bigquery = BigQueryOptions.defaultInstance().service();
 TableId tableId = TableId.of("dataset", "table");
-TableInfo info = bigquery.getTable(tableId);
-if (info == null) {
+Table table = bigquery.getTable(tableId);
+if (table == null) {
```

Comment:
Yes, it is just an example and it is readable.
However, I do think that the propose change is better for the following reasons:
1) slightly shorter
2) save some code indentation
3) more logical flow (and therefore example). create a table if does not exists and populate with data rather than either create a table if not exists or put data if exists.

Yes, if we decide to change it here we should also change it in other places were we apply a similar pattern.
We do not have to do it in this PR or at all if you disagree with my reasons.